### PR TITLE
Kickoff preparation minutes

### PR DESCRIPTION
**Pull request by ineiti**
My minutes from the kickoff preparation meeting.

### CODE REVIEW

File: content/post/2024-03-12_Kickoff-preparation.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
++++
+title = 'Kickoff Preparation'
+date = 2024-03-12T16:19:21+01:00
+draft = true
```
Comment (elanti) [inline content/post/2024-03-12_Kickoff-preparation.md]: Should this be set to `false` to have it published?
```suggestion
draft = false
```

File: content/_index.html [hunk 2]
```diff
@@ -48,12 +48,21 @@ <h1>Participating</h1>
 
     <p>Looking forward to seeing you!</p>
 
-    <h2>Upcoming meetings</h2>
+    <h2>Upcoming posts</h2>
```
Comment (elanti) [inline content/_index.html]: I would prefer to keep "meetings" as it indicates to the reader that there is an actual gathering. As a side note, for future meetings, we could also add the location and Zoom link (I will make a PR to update the "Kickoff meeting"). 
Once the meeting is passed and we write the minute of it, it is then placed into the "post" category.

File: content/_index.html [hunk 3]
```diff
@@ -48,12 +48,21 @@ <h1>Participating</h1>
...
     <ul>
-        <li>2024-03-05 - <a href="/post/2024-02-08_preparation">Kickoff preparation meeting</a></li>
         <li>2024-05-07 - Kickoff meeting</li>
     </ul>
 
+    <h2>Active posts</h2>
```
Comment (elanti) [inline content/_index.html]: Just to understand the logic, what do you mean by "active post"? Posts that will be updated and that will be placed into "Past posts" once finalized?

**Review by elanti — COMMENTED** [review]
Thanks @ineiti, it looks great ! 
I just have a few minor comments and questions.

File: content/post/2024-03-12_Kickoff-preparation.md (hunk 1 above)
Comment (ineiti) [reply content/post/2024-03-12_Kickoff-preparation.md]: Yes, absolutely. The normal use-case is if you have articles in preparation in your main branch. Which is not the case here, as I work in a separate branch. So probably it is safe to put this to `false` from the beginning...

File: content/_index.html (hunk 2 above)
Comment (ineiti) [reply content/_index.html]: It depends on how we want to structure the page. I thought we might put other posts up, too. Like referencing meetings from ETHZ, updates on the kickoff preparation, perhaps other interesting articles.

What do you think?

File: content/_index.html (hunk 3 above)
Comment (ineiti) [reply content/_index.html]: Might be overkill: `Active posts` == `Recent posts`. I might be overthinking this :)

I would like to have the following types of posts:
- upcoming meetings
- non-EPFL meetings (ETHZ, UK, France?)
- general posts (update on the grant proposal, if somebody wants to present an interesting library/application, ...)
- minutes of past meetings
- past meetings

Any good ideas how to present this in a nice way?

File: content/_index.html (hunk 2 above)
Comment (elanti) [reply content/_index.html]: Ok, that makes sense to me !

File: content/_index.html (hunk 3 above)
Comment (elanti) [reply content/_index.html]: Ok I understand ! Maybe we can then have only the "Upcoming Posts" and "Posts" sections to simplify a bit.

For the different kinds of posts, maybe we could use tags or categories. All the posts will appear, but you can also sort them by tags/categories depending on your interest.

In the future, when the project gets bigger, some of these categories may need a dedicated page.

Comment (ineiti) [reply content/_index.html]: `Upcoming Posts` -> `Upcoming meetings`?
`Posts` - contain past meetings, minutes, and other posts?

Comment (elanti) [reply content/_index.html]: Yes, perfect for me!

File: content/post/2024-03-12_Kickoff-preparation.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+We also decided for the following tasks.
+If one of them picks your interest, don't hesitate to reach out to the persons.
+Or, even better, join the matrix discussion at [#rse-epfl:epfl.ch](https://matrix.c4dt.org/#/room/#rse-epfl:epfl.ch)
+and jump in.
+
+- Pre-event organization
+  - Poll for signup to the event: Ahmed
+  - Announcements on EPFL screens and in memento: Linus
+  - Creating flyer / nice page for screens: Nicolas
```
Comment (PascalinDe) [inline content/post/2024-03-12_Kickoff-preparation.md:29]:
```suggestion
  - Creating flyer / nice page for screens: Nicolas
  - Gathering topics for topical discussions: Carine, Valérian
```

**Comment by elanti** [comment]
LGTM, thanks for the work ! Should I merge it or wait for the two assigned reviewers ?

**Comment by ineiti** [comment]
> LGTM, thanks for the work ! Should I merge it or wait for the two assigned reviewers ?

I wrote on matrix that I'll wait till the end of today for other input, then I'll merge and publish it.

File: content/post/2024-03-12_Kickoff-preparation.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+  - Gathering topics for topical discussions: Carine, Valérian
+- During event:
+  - MC: Linus 
+  - Zoom handler: Son Pha
```
Comment (sphamba) [inline content/post/2024-03-12_Kickoff-preparation.md]:
```suggestion
  - Zoom handler: Son
```

File: content/_index.html [hunk 6]
```diff
@@ -50,10 +50,15 @@ <h1>Participating</h1>
...
-        <li>2024-03-05 - <a href="/post/2024-02-08_preparation">Kickoff preparation meeting</a></li>
         <li>2024-05-07 - Kickoff meeting</li>
     </ul>
 
+    <h2>All Posts</h2>
+    <ul>
+        <li>2024-03-12 - <a href="/post/2024-03-12_kickoff-preparation">Summary of the preparation meeting</a></li>
+        <li>2024-03-05 - <a href="/post/2024-02-08_preparation">Kickoff preparation meeting</a></li>
+    </ul>
+
```
Comment (sphamba) [inline content/_index.html:61]: With Hugo, it should be possible to automatically list all posts. I don't know how to do it though

**Review by sphamba — APPROVED** [review]

File: content/_index.html (hunk 6 above)
Comment (ineiti) [reply content/_index.html:61]: #3

**Pull request merged**